test/crawler_test.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import time
+
+def crawler_test(uid, upwd):
+    auth = (uid, upwd)
+    base = 'http://localhost:8000/manager/crawlers/'
+    data = {'year':'2018', 'semester':'1학기'}
+
+    # I don't know the reason, but it works only if requests is imported here.
```

Comment:
Didn't I answered why this condition was occurred?
If so, you have to remove these comment lines.